### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Extracted from https://github.com/digitalbazaar/http-signature-header/pull/19

Extracted from https://github.com/digitalbazaar/http-signature-zcap-invoke/blob/master/main.js#L35

WIP, todo:
- [x] Find an isomorphic way to base64-encode strings (our `base64url-universal` does not expose non-url encoding)
- [x] ~~Find some test strings for multihash digest?~~ Opened issue #2.

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
Switched to using universal base64 and base64url libs. Except that something is off with the karma tests, giving me

```
AssertionError: expected 'SHA-256=OTUsMTQzLDQsMjQ2LDE2MywxNjgsMTQ2LDE3MCwxNzEsMTg5LDIxOSwxMDgsMjQyLDExNSwxMzcsNjgsMTQ3LDExOSw1Nyw5NiwyMTIsMTYzLDM3LDE3Nyw1LDI1NCwyMjgsMTEwLDIzOSw2Nyw0LDI0MQ==' to equal 'SHA-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE='
```

Investigating.

**Comment by OR13** [comment]
please merge.

File: lib/base64.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
```
Comment (dlongley) [inline lib/base64.js:4]: Here and elsewhere: (ESM doesn't need `use strict`):

```suggestion
```

File: lib/base64.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline lib/base64.js:2]: Here and elsewhere:

```suggestion
 * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
```

**Review by dlongley — APPROVED** [review]
Minor nits to clean up. Also, tests should cover multihash version, particularly if we are to replace what's in http-signature-zcap-invoke with this.

**Comment by dmitrizagidulin** [comment]
Thanks @dlongley, will clean those up now..

File: lib/base64.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/base64.js]: Fixed, thanks.

File: lib/base64.js (hunk 2 above)
Comment (dmitrizagidulin) [reply lib/base64.js]: Fixed.

**Comment by dmitrizagidulin** [comment]
@OR13 @dlongley also renaming this repo to `http-digest-header` (since now Digest header has its own spec).

**Pull request merged**